### PR TITLE
Fixes problems with test suites that print on import, also fixes problems with test suites that use setUpClass

### LINKED ISSUE #8: Suite finding doesn't work if the test discovery includes printed text
If you have any sort of code that prints to stdout on import of a test file, it is read as a test suite and added to the tests (and subsequently breaks tests.)

### PR DESCRIPTION
**Pull request by jakegt1**
Fixes #8 #9 

I will note that i do not write typescript, so if my code is not sufficient please suggest a change.

### CODE REVIEW

File: src/unittestScripts.ts [hunk 1]
```diff
@@ -27,23 +27,22 @@ def get_tests(suite):
         return [suite]
 
 
-def discover_tests():
+def discover_with_loader():
```
Comment (kondratyev-nv) [inline src/unittestScripts.ts]: How about renaming it to something like `discover_suits`? IMO it better describes this method.

File: src/unittestScripts.ts [hunk 2]
```diff
@@ -27,23 +27,22 @@ def get_tests(suite):
...
 
 
 def run_tests(test_names):
-    tests = load_tests(test_names)
     runner = TextTestRunner(resultclass=TextTestResultWithSuccesses)
-    results = [runner.run(test) for test in tests]
+    if not test_names:
+        results = [runner.run(discover_with_loader())]
+    else:
+        results = [runner.run(loader.loadTestsFromName(name)) for name in test_names]
```
Comment (kondratyev-nv) [inline src/unittestScripts.ts]: Maybe I'm missing something, but isn't `loader` defined in this scope?

File: src/unittestScripts.ts (hunk 1 above)
Comment (jakegt1) [reply src/unittestScripts.ts]: yeah that's fair enough - i will note though that it's `suites` and not `suits` (i didn't take the liberty of changing the misspelling on the typescript)

File: src/unittestScripts.ts (hunk 2 above)
Comment (jakegt1) [reply src/unittestScripts.ts]: Hmm, unsure how i didn't notice this. Will go fix.

**Pull request merged**